<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>230</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift Light</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Open Image</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Color Effects</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>214</width>
             <height>204</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QRadioButton" name="vintage">
              <property name="text">
               <string>Vintage</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="frozen">
              <property name="text">
               <string>Bluelight</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="casting">
              <property name="text">
               <string>Casting</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="blue">
              <property name="text">
               <string>Blue</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="red">
              <property name="text">
               <string>Red</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="green">
              <property name="text">
               <string>Green</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="comic">
              <property name="text">
               <string>Comic</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="reverse">
              <property name="text">
               <string>Reverse</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="colorCheck">
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>-74</y>
             <width>214</width>
             <height>204</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QRadioButton" name="lightening">
              <property name="text">
               <string>Ligthening</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="vignetting">
              <property name="text">
               <string>Vignetting</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="leak1">
              <property name="text">
               <string>Leak 1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="leak2">
              <property name="text">
               <string>Leak 2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="leak3">
              <property name="text">
               <string>Leak 3</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="leak4">
              <property name="text">
               <string>Leak 4</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="leak5">
              <property name="text">
               <string>Leak 5</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="noise">
              <property name="text">
               <string>Noise</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="filterCheck">
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>266</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="open"/>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>Filter</string>
    </property>
    <addaction name="color"/>
    <addaction name="special"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFilter"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="open">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="color">
   <property name="text">
    <string>Color Effects</string>
   </property>
  </action>
  <action name="special">
   <property name="text">
    <string>Special Effects</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
